docs/usage_guide/advanced_usage/index.rst

```diff
@@ -30,42 +31,54 @@ Modin also supports these experimental APIs on top of pandas that are under acti
+functions, and create a DataFrame from those modified partitions. Visit
+:doc:`pandas partitioning API </flow/modin/distributed/dataframe/pandas>` documentation to learn more.
+
 Modin Spreadsheet API: Render Dataframes as Spreadsheets
 --------------------------------------------------------
-The Spreadsheet API for Modin allows you to render the dataframe as a spreadsheet to easily explore 
-your data and perform operations on a graphical user interface. The API also includes features for recording 
-the changes made to the dataframe and exporting them as reproducible code. Built on top of Modin and SlickGrid, 
-the spreadsheet interface is able to provide interactive response times even at a scale of billions of rows. 
+
+The Spreadsheet API for Modin allows you to render the dataframe as a spreadsheet to easily explore
+your data and perform operations on a graphical user interface. The API also includes features for recording
+the changes made to the dataframe and exporting them as reproducible code. Built on top of Modin and SlickGrid,
+the spreadsheet interface is able to provide interactive response times even at a scale of billions of rows.
 See our `Modin Spreadsheet API documentation`_ for more details.
 
-.. figure:: ../img/modin_spreadsheet_mini_demo.gif
+.. figure:: /img/modin_spreadsheet_mini_demo.gif
    :align: center
    :width: 650px
-   :height: 350px   
+   :height: 350px
 
 Progress Bar
 ------------
-Visual progress bar for Dataframe operations such as groupby and fillna, as well as for file reading operations such as 
+
+Visual progress bar for Dataframe operations such as groupby and fillna, as well as for file reading operations such as
 read_csv. Built using the tqdm library and Ray execution engine. See `Progress Bar documentation`_ for more details.
 
-.. figure:: ../img/progress_bar_example.png
-   :align: center   
+.. figure:: /img/progress_bar_example.png
+   :align: center
 
 Dataframe Algebra
 -----------------
+
 A minimal set of operators that can be composed to express any dataframe query for use in query planning and optimization.
 See our `paper`_ for more information, and full documentation is coming soon!
 
 SQL on Modin Dataframes
 ------------------------
 
-.. figure:: ../img/modin_sql_example.png
-   :align: center   
+.. figure:: /img/modin_sql_example.png
+   :align: center
 
 Read about Modin Dataframe support for SQL queries in this recent `blog post`_. Check out the `Modin SQL documentation`_ as well!
 
 Distributed XGBoost on Modin
 ----------------------------
+
 Modin provides an implementation of distributed XGBoost machine learning algorithm on Modin DataFrames. See our
 :doc:`Distributed XGBoost on Modin documentation <modin_xgboost>` for details about installation and usage, as well as
 :doc:`Modin XGBoost architecture documentation </flow/modin/experimental/xgboost>` for information about implementation and
```

Comment:
link to this blogpost? 
https://medium.com/intel-analytics-software/distributed-xgboost-with-modin-on-ray-fc17edef7720

Reply:
added